content/blog/2020-05-26-may-20-community-gems.md

```diff
@@ -0,0 +1,93 @@
+---
+title: May '20 Community Gems
+date: 2020-05-26
+description: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  development best practices, sharing models and data across projects, 
+  and using DVC with teams.
+descriptionLong: |
+  A roundup of technical Q&A's from the DVC community. This month, we discuss 
+  development best practices, sharing models and data across projects, 
+  and using DVC with teams.
+picture: 2020-05-26/May_20_Gems_Header.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/april-20-community-gems/356
+tags:
+  - Discord
+  - Gems
+  - Cache
+  - Google Cloud Storage
+  - Import
+---
+
+## Discord gems
+
+Here are some Q&A's from our Discord channel that we think are worth sharing.
+
+### Q: [How do I completely delete a file from DVC?](https://discord.com/channels/485586884165107732/563406153334128681/710546561498873886)
+
+You can simply delete a `.dvc` file to remove its target from DVC tracking, and
+then run `dvc gc` to
+[clear your DVC cache](https://dvc.org/doc/command-reference/gc#gc). If you want
+to remove a single `.dvc` file without doing a cache cleanup, look into the
+`.dvc` file and note the `md5` field inside. Then use this value to identify the
+corresponding file in your DVC cache and delete it. For example: if your target
+file has `md5: 123456`, the corresponding file in your cache will be
+`.dvc/cache/12/3456`.
+
+In the future, we plan to support a function like `git rm` that will allow for
+more granular delete options.
+
+### Q: [I'm using an S3 bucket as a DVC remote; the bucket is shared with several coworkers, who have projects in other folders in the bucket. Will it disrupt DVC if I manually add a README inside the folder being used for remote storage, to let my coworkers know what the directory is used for?](https://discord.com/channels/485586884165107732/563406153334128681/707551737745244230https://discord.com/channels/485586884165107732/563406153334128681/707551737745244230)
```

Comment:
may be we can generalize it a bit - Is it safe to add custom file to remote storage? or something like this? And put the long version at the beginning of the answer (like E.g. I want to add REAME to describe the content ...)

Reply:
changed to: 
```
### Q: [Is it safe to add a custom file to my DVC remote?](https://discord.com/channels/485586884165107732/563406153334128681/707551737745244230https://discord.com/channels/485586884165107732/563406153334128681/707551737745244230)

Definitely. Some people add additional files to their DVC remote, like a README to explain to teammates what the folder is being used for. 
Having an additional file in the remote that isn't part of
DVC tracking won't pose any issues. You would only encounter problems if you
were manually modifying or deleting contents of the remote managed by DVC.
```